lte/gateway/python/scripts/dp_actions_cli.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
```

Comment:
lets rename it to dp-probe cli, or something else that is not very specific, since we will continue to add more features to this cli.